lib/traject/yale_csv_config.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+# Yale CSV Mapping Configuration
+require_relative 'csv_reader'
+require_relative 'dlme_json_resource_writer'
+require_relative 'macros/dlme'
+require_relative 'macros/csv'
+extend Macros::DLME
+extend Macros::Csv
+
+settings do
+  provide 'writer_class_name', 'DlmeJsonResourceWriter'
+  provide 'reader_class_name', 'CsvReader'
+end
+
+to_field 'agg_provider',     provider
+to_field 'id',               normalize_prefixed_id('OrbisBibID')
+to_field 'cho_identifier',   column('OrbisBibID', trim: true)
+to_field 'cho_provenance',   column('Place of origin', trim: true)
```

Comment:
This seems like place of publication information. Is this the right mapping?

Reply:
The values are tricky here - there is analysis on the values in the spreadsheet that support the odd mapping (if i recall correctly).